bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml

```diff
@@ -246,6 +247,18 @@ limitations under the License.
                   <pattern>org.apache.commons.codec</pattern>
                   <shadedPattern>com.google.bigtable.repackaged.org.apache.commons.codec</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>org.threeten</pattern>
+                  <shadedPattern>com.google.bigtable.repackaged.org.threeten</shadedPattern>
+                </relocation>
+                <relocation>
+                  <pattern>org.checkerframework</pattern>
+                  <shadedPattern>com.google.bigtable.repackaged.org.checkerframework</shadedPattern>
```

Comment:
I'm a little concerned about shading the checkerframework. I dont really know how it works, but I'm guessing it uses annotations with some kind of tool that does static analysis. If my understanding is correct then shading it would break the static analysis. 
Is this dependency coming from gcj? I'm wondering how it's used.

Reply:
I think exclude it.

Reply:
grpc added the `checkerframework` dependency.  I'll ask the diamond dependency group about this.

Reply:
Hmm... Guava has it as well.  I'll exclude it from shading, and manually promote the dependency.

Reply:
For future: I wonder if it can be demoted to the provided scope? If either of those projects only use it the annotations, then we should send PRs to both to change the scope and avoid dealing with it in the client